lib/rubocop/cop/lint/void.rb

```diff
@@ -80,6 +80,12 @@ class Void < Cop
                                  swapcase tr tr_s transform_values
                                  unicode_normalize uniq upcase].freeze
 
+        def on_block(node)
+          return unless node.body && node.body.single_line?
```

Comment:
This is technically incorrect. It has nothing to do with the number of lines. It's the number of expressions.

E.g.:

```
ruby-parse -e 'foo { bar; baz }'
(block
  (send nil :foo)
  (args)
  (begin
    (send nil :bar)
    (send nil :baz)))
```

Reply:
So I guess this should be:

```ruby
return unless node.body && !node.body.begin_type?
```